.github/workflows/generate.yml

```diff
@@ -0,0 +1,55 @@
+name: Regenerate Asset Data and PR
+
+on:
+  workflow_dispatch:
+  pull_request:
+  # schedule:
+  #   - cron: '0 0 * * *' # Schedule to run automatically, uncomment for production use
+
+jobs:
+  regenerate-and-create-pr:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.ref == 'feat_regenerate_asset_data_gh_action'
```

Comment:
To be reverted too before opening obviously, in profit of the cron schedule above, and without the `pull_request:` directive